apps/src/javalab/javalabRedux.js

```diff
@@ -100,6 +102,13 @@ export const setSource = (
   isValidation
 });
```

Comment:
Nice find! Do we even need this `setSource` action creator anymore? Are we using the `isVisible` and `isValidation` arguments anywhere?

Reply:
I'm not seeing any use of the `isVisible` or `isValidation` arguments anywhere, although we do still use the `setSource` action creator (with only two arguments) in a couple of places (new file, import of file).

I found at least one unnecessary import of `setSource` in `Backpack`, so just removed that in most recent commit.